package.json

```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "async": "1.5.2",
-    "augur-contracts": "3.3.10",
+    "augur-contracts": "^3.4.0-1",
```

Comment:
Caret version here on purpose allows us to NPM Upgrade on the dev channel releases